Sources/NostrSDK/Bech32IdentifierType.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Bech32IdentifierType.swift
+//
+//
+//  Created by Terry Yiu on 6/30/24.
+//
+
+/// The type of Bech32-encoded identifier.
+/// These identifiers can be used to succinctly encapsulate metadata to aid in the discovery of events and users.
+/// See [NIP-19](https://github.com/nostr-protocol/nips/blob/master/19.md) for information about how these Bech32-encoded
```

Comment:
I see that this was copied from _MetadataCoding.swift_, but the end of this line doesn't finish the sentence.